locales/en/messages.json

```diff
@@ -1401,6 +1401,13 @@
         "message": "Preview"
     },
 
+    "receiverMspWarningText": {
+        "message": "These sticks allow BetaFlight to be armed and tested without a transmitter or receiver being present. However, <strong>this feature is not intended for flight and propellers must not be attached.</strong><br><br>This feature does not guarantee reliable control of your craft. <strong>Serious injury is likely to result if propellers are left on.</strong>"
```

Comment:
The commonly used for is 'Betaflight', not 'BetaFlight'.

Reply:
True. This was the original text. I will modify it after the weekend. I'm out.